tests/end-to-end/deprecation-trigger/_files/deprecation-trigger-class/vendor/autoload.php

```diff
@@ -0,0 +1,12 @@
+<?php declare(strict_types=1);
+
+spl_autoload_register(function ($class) {
```

Comment:
This reflects what happens when loading classes usually, if we do the same as in #5844 we'll trigger the deprecation during the `loadBootstrapScript`.